parsec/core/gui/file_items.py

```diff
@@ -44,7 +48,9 @@ class IconTableItem(CustomTableItem):
     UNSYNCED_ICON = None
     CONFINED_ICON = None
 
-    def __init__(self, is_synced, is_confined, pixmap, *args, **kwargs):
+    def __init__(
+        self, is_synced: bool, is_confined: bool, pixmap: QPixmap | str, *args: Any, **kwargs: Any
```

Comment:
`pixmap` can be a `QPixmap` here? It seems to only ever be a str.

Reply:
Done.